fitbenchmarking/fitting/mantid/func_def.py

```diff
@@ -41,45 +44,106 @@ def function_definitions(problem):
 
     problem_type = extract_problem_type(problem)
```

Comment:
Do we need this anymore? We check the instance of the function and then by default use parse_function_definitions function

Reply:
Same goes for the scipy/func_def.py